kong/tools/module.lua

```diff
@@ -0,0 +1,27 @@
+local find = string.find
```

Comment:
We should also localize `type` `error` `xpcall`

Reply:
done